myapp/templates/index.html

```diff
@@ -145,6 +145,22 @@
         xhr.setRequestHeader('X-CSRFToken', getCookie('csrftoken'));
         xhr.send();
     }
+
+    setInterval(function() {
+        $.ajax({
+            method: "GET",
+            url: "",
+            success: function(data) {
+            var parser = new DOMParser();
```

Comment:
as kamal commented, this way would cause a problem on the table not clickable, because a new table inserted, click event handlers on rows